### ISSUE TITLE
Some link related 

### ISSUE THREAD

**Issue by HumanG33k** [CellularPrivacy/Android-IMSI-Catcher-Detector#961]
### Agreements

- [ x] **I have reviewed and accepted the [guidelines for contributing](https://github.com/CellularPrivacy/Android-IMSI-Catcher-Detector/blob/development/.github/CONTRIBUTING.md) to this project.**
- [ x] **I have searched the [open Issues](https://github.com/CellularPrivacy/Android-IMSI-Catcher-Detector/issues) and made sure I am not filing a duplicate.**
- [ x] **I have read the notes on [debugging](https://github.com/CellularPrivacy/Android-IMSI-Catcher-Detector/blob/development/.github/CONTRIBUTING.md#debugging) and will properly maintain this Issue.**

### Overview

https://thehackernews.com/2019/02/location-tracking-imsi-catchers.html

**Comment by beerisgood** [comment]
Yes, we need a rebuild for this app which works and work against such stuff
